string detector_node
string message_type
int32 count
